cmd/nigiri/resources/docker-compose.yml

```diff
@@ -108,7 +108,7 @@ services:
     environment:
       API_URL: http://localhost:3000
     ports:
-      - 5000:5000
+      - 4400:5000
```

Comment:
Revert to default ports (you can edit this on your data directory and/or soon via ENV variable #145 )